lib/rubycritic/analysers_runner.rb

```diff
@@ -25,13 +24,15 @@ def initialize(paths)
 
     def run
       ANALYSERS.each { |analyser| analyser.new(analysed_modules).run }
+      puts "The (limited) average cost is: #{analysed_modules.average_cost}"
```

Comment:
Similar feelings from https://github.com/whitesmith/rubycritic/pull/36#issuecomment-158670848 - the output shouldn't be inside the `AnalysersRunner` class.